images/linux/scripts/installers/1604/preparemetadata.sh

```diff
@@ -7,6 +7,7 @@
 
 source $HELPER_SCRIPTS/document.sh
 
+WriteItem "<!--- This file is autogenerated, it is strongly recommended not to modify it. -->"
```

Comment:
I think this message can be more direct, something like:
```
DO NOT EDIT - This markdown file is autogenerated
```

Reply:
I have added the string according to provided example.